app/views/includes/_main_footer.html.erb

```diff
@@ -2,21 +2,22 @@
 <footer class="text-center">
   <!-- Links -->
   <div class="footerBar comtainer ">
```

Comment:
Hey @kholtzman5610 the class `comtainer` looks like mispelling. Can you try removing it and see if it still looks ok? If it does let's just remove that class.